src/index.js

```diff
@@ -1,4 +1,60 @@
 import _ from "lodash";
 import "./style.css";
 
-console.log("Hello, from Webpack!");
+const baseUrl = "https://us-central1-js-capstone-backend.cloudfunctions.net/api";
+const gameId = "dxIcv0VBs2ScW6iHCjNM";
+
+// Get all scores
+const getScroes = async () => {
+  const scoreResponse = await fetch(`${baseUrl}/games/${gameId}/scores/`, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  });
+  const scoreData = await scoreResponse.json();
+  return scoreData;
+};
+
+// Add score for user
+const postScroes = async (userName, userScore) => {
+  const data = {
+    user: userName,
+    score: userScore
+  };
+  const scoreResponse = await fetch(`${baseUrl}/games/${gameId}/scores/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(data)
+  });
+  const scoreData = await scoreResponse.json();
+  alert(scoreData.result);
```

Comment:
As a requirement that we maintain [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), I suggest you use a message below the form or something else to display the feedback or validation to the user😉. Since it is impossible to style and stop code execution😉.
<img width="491" alt="image" src="https://github.com/Aqurds/mv-leaderboard/assets/64643995/a71f8a13-bfb4-440b-af74-75ba91e63c9c">


<img width="743" alt="image" src="https://github.com/Aqurds/mv-leaderboard/assets/64643995/5715d60a-c934-463d-b809-c4c6c83638fe">